integration-tests/features/steps/cockpit_demo.py

```diff
@@ -36,33 +36,27 @@ def visit_demo_page(context, menu_item):
     session.login()
     session.open_plugin(menu_item)
 
-@then("the local VMs should be listed within {time_limit:g} seconds")
-def check_demo_machine_listing(context, time_limit):
-    """Checks contents and response time for the demo plugin"""
+@when("enters {source_vm}'s IP address as the import source")
+def step_impl(context, source_vm):
     session = context.demo_session
-    expected_machines = context.vm_helper.machines
-    expected_hosts = sorted(host for name, host in expected_machines.items())
-    missing = session.list_missing_machines(expected_hosts, time_limit)
-    assert_that(missing, equal_to([]), "Machines missing from listing")
+    source_ip = context.vm_helper.get_ip_address(source_vm)
+    session.enter_source_ip(source_ip)
+    session.wait_for_active_import_button(10)
 
-@when("the demonstration user redeploys {source_vm} to {target_vm}")
-def redeploy_vm_via_plugin(context, source_vm, target_vm):
+@when('clicks the "Import" button')
+def import_app_via_plugin(context):
     session = context.demo_session
-    gethost = context.vm_helper.get_hostname
-    session.start_vm_redeployment(gethost(source_vm), gethost(target_vm))
-    session.wait_for_redeployment_to_start(10)
+    session.start_app_import()
+    input("Hit enter to continue testing")
```

Comment:
This is for debugging, since it pauses the test while the failed command is displayed in the browser.